### DISCUSSION TITLE
How can I display special characters?

### DISCUSSION THREAD

**Discussion by mauroporras** [tpope/vim-fugitive#1711] [Q&A] [answered]
Hello there,

This is what I get inside nvim:

<img width="589" alt="Screen Shot 2021-03-24 at 07 24 55" src="https://user-images.githubusercontent.com/167700/112309983-09974100-8c72-11eb-9ad7-656c7015e27f.png">

Is there a way to properly display special chars? I already disabled all my plugins, except this one.

Thank you.

**Comment by tpope** [comment, marked as answer]
Those are ANSI escape sequences for turning on color. Fugitive tries quite aggressively to turn that off, although apparently not aggressively enough. I will fix this, but in the meantime, I recommend you do _not_ set `color.ui=always`. "Always" means "do it anyways even in places it won't work", and Fugitive is one of those places.
